frontend/js/src/user/components/PinnedRecordingCard.tsx

```diff
@@ -148,6 +151,21 @@ export default class PinnedRecordingCard extends React.Component<
           action={() => this.unpinRecording()}
         />
       );
+      additionalMenuItems.push(
+        <ListenControl
+          text="Edit Comment"
+          key="Edit Comment"
+          icon={faPencilAlt}
+          action={() => {
+            NiceModal.show(PinRecordingModal, {
+              recordingToPin: listen,
+              rowId: pinnedRecording.row_id,
+            });
```

Comment:
There is a good example in the codebase of how to use that return value in the PinnedRecordingCard component: 
https://github.com/metabrainz/listenbrainz-server/blob/c8b5abf56ba77b3b1c81465b01e51b0e144ac6e1/frontend/js/src/user/playlists/Playlists.tsx#L171-L175

That also means you will have to store `pinnedRecording` in the component state of PinnedRecordingCard rather than using it from the props.

You'll need a new class method `updatePinnedRecording` or something like that where you update the state accordingly, similar to what we do here:
https://github.com/metabrainz/listenbrainz-server/blob/c8b5abf56ba77b3b1c81465b01e51b0e144ac6e1/frontend/js/src/user/playlists/Playlists.tsx#L96-L101

Except since pinnedRecording is an object we want to merge properties, something like:
```
    this.setState( currentState => ({
      pinnedRecording: {
        ...currentState.pinnedRecording,
        blurb_content: updatedBlurbContent},
    }));
```